### DISCUSSION TITLE
Hyperparameter tuning for unsupervised outlier detection

### DISCUSSION THREAD

**Discussion by meiyun1995** [yzhao062/pyod#524] [General]
i'm using the pyod's CBLOF algorithm for unsupervised outlier detection. There is an alpha and beta hyperparameter to tune. Since it is unsupervised, I don't have any labels to measure the performance with the different hyperparameter. Is there a way I can tune this using pyod? I see a paper on this Hyperparameter tuning for unsupervised outlier detection but it doesn't use CBLOF https://arxiv.org/abs/2208.11727

**Comment by yzhao062** [comment]
unsupervised hp tuning is still in its early stage and not used in pyod yet...
